# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: eaa23cb3b396
# Full Hash: eaa23cb3b39686e114b1e78a78c2f8e5c91bcb29
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2019-12-28 04:06:02
# Regressor Bug: 1605471
# File Overlap Count: 5
# Description:
#   Bug 1605471 - Update cubeb-coreaudio to acb90e9. r=alwu
#   
#   Pick commits:
#   acb90e9 - Add license for mixer crate
#   82e7ff9 - Move criterion crate to [dev-dependencies]
# ==============================================================================

diff -r 509cd3b3ec22 -r eaa23cb3b396 .cargo/config.in
--- a/.cargo/config.in	Fri Dec 27 16:05:26 2019 +0000
+++ b/.cargo/config.in	Fri Dec 27 16:58:18 2019 +0000
@@ -65,7 +65,7 @@
 [source."https://github.com/ChunMinChang/cubeb-coreaudio-rs"]
 git = "https://github.com/ChunMinChang/cubeb-coreaudio-rs"
 replace-with = "vendored-sources"
-rev = "868d847c2e95096b6eec629dfed77ef363363ecb"
+rev = "acb90e9bf36e6e035ac6bbe51efa0a8825b5b6be"
 
 [source.crates-io]
 replace-with = "vendored-sources"